datadog-crashtracker/src/crash_info/telemetry.rs

```diff
@@ -497,13 +517,13 @@ mod tests {
             .set_host_from_url(&format!("file://{}", output_filename.to_str().unwrap()))
             .unwrap();
 
-        let crash_uuid = "test-uuid-12345";
+        let crash_uuid = "19ea82a5-2118-4fb0-b0dd-6c067a3026c6";
```

Comment:
This will have to change if you remove the ability to set, instead grab it from the builder